src/js/navbar.js

```diff
@@ -10,7 +10,10 @@ var NavBar = props => (
     <input id='bmenub' type='checkbox' class='show'/>
     <label htmlFor='bmenub' class='burger pseudo button'>&#8801;</label>
     <div class='menu'>
+      <span>
+      <a>{auth.name}</a>
```

Comment:
What is this? Just do `<span>{auth.name}</span>`. Doesnt work?. Then close the span before the Link to uestions at least